Test failure on 32-bit

When trying to run the tests on i386, one of them fails apparently due to an overflow:

```
# github.com/hydrogen18/stalecucumber
src/github.com/hydrogen18/stalecucumber/pickle_writer_test.go:132: constant 18446744073709551615 overflows uint
FAIL	github.com/hydrogen18/stalecucumber [build failed]
?   	github.com/hydrogen18/stalecucumber/struct_export_test	[no test files]
```
Please see a full log at Debian's reproducible builds pages: https://tests.reproducible-builds.org/debian/rb-pkg/unstable/i386/golang-github-hydrogen18-stalecucumber.html

Fri Oct 28 22:27:23 UTC 2016  I: starting to build golang-github-hydrogen18-stalecucumber/unstable/i386 on jenkins.debian.net on '2016-10-28 22:27'
Fri Oct 28 22:27:23 UTC 2016  I: The jenkins build log is/was available at https://jenkins.debian.net/job/reproducible_builder_i386_1/20997/console
Fri Oct 28 22:27:23 UTC 2016  I: Downloading source for unstable/golang-github-hydrogen18-stalecucumber
--2016-10-28 22:27:25--  http://ftp.de.debian.org/debian/pool/main/g/golang-github-hydrogen18-stalecucumber/golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.dsc
Resolving localhost (localhost)... 127.0.0.1
Connecting to localhost (localhost)|127.0.0.1|:3128... connected.
Proxy request sent, awaiting response... 200 OK
Length: 2456 (2.4K)
Saving to: ‘golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.dsc’

     0K ..                                                    100%  212M=0s

2016-10-28 22:27:25 (212 MB/s) - ‘golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.dsc’ saved [2456/2456]

Fri Oct 28 22:27:25 UTC 2016  I: golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.dsc
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Format: 3.0 (quilt)
Source: golang-github-hydrogen18-stalecucumber
Binary: golang-github-hydrogen18-stalecucumber-dev
Architecture: all
Version: 0.0~git20161012.0.cd9ec28-1
Maintainer: Debian Go Packaging Team <pkg-go-maintainers@lists.alioth.debian.org>
Uploaders: Sascha Steinbiss <satta@debian.org>
Homepage: https://github.com/hydrogen18/stalecucumber
Standards-Version: 3.9.8
Vcs-Browser: https://anonscm.debian.org/cgit/pkg-go/packages/golang-github-hydrogen18-stalecucumber.git
Vcs-Git: https://anonscm.debian.org/git/pkg-go/packages/golang-github-hydrogen18-stalecucumber.git
Build-Depends: debhelper (>= 9), dh-golang, golang-go
Package-List:
 golang-github-hydrogen18-stalecucumber-dev deb devel extra arch=all
Checksums-Sha1:
 0b0bb22a3e9561c34da66b8faf36e6ce1b420de3 28580 golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28.orig.tar.xz
 b11b5055ffcade826f044e5eae34a8fb042571e4 1896 golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.debian.tar.xz
Checksums-Sha256:
 078b378233dd3b432fa28ef7949e55bd3b1080abb766fb29c3a87d843e83da4b 28580 golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28.orig.tar.xz
 6143bd886f78b9a09b4e6ad8e1e37aba09caf13c04ef20a6b4dd2314cebe22cc 1896 golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.debian.tar.xz
Files:
 1633ed3dcdb2d1578a199aa93235fe8e 28580 golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28.orig.tar.xz
 326ef26d45e8c873c045e64284df04dc 1896 golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.debian.tar.xz
Go-Import-Path: github.com/hydrogen18/stalecucumber

-----BEGIN PGP SIGNATURE-----

iQIcBAEBCAAGBQJYD2GNAAoJEOjTeuLwn0hyj9UP/0YZJCiuiH0L6MNnLkrRO8LS
SdHgm9svY3h5aPsi9BQE6el8spocdmU9jdLX5Y2kAS0FxrbrFcErjEbXw4bdjnXW
CdLtGg52vzj9sMELVfmFeoeeICd73f645hNTriSmnZ8OMghQHzM3Xm+Ps94K//lm
TIZ3IilfaeY23yHHgANNjbeWc0D5E3dD6wsgF5PGvaLVxRQHjV2H9kmPwcOrrFwI
T3eniDZSRJPJfzD3vA+67q8+3VSuw2A7du7WP7keWokRYhnJZDdEIvKqiJ49+5mb
zwyIRWq1s0w1MebXQtVd5hTkQGpdL05791/4OwxWQ5vMjm05qFoSdIooP9I+dXIG
oonAsK4FKSRIUo0P31f0AqTx7AfptU+MbdrftKWaI0kM1jZQhrkFhM6/Wbz6XYys
0am+42FGarLbbmpzMa9ec6rAttpkojbWIvfzbymEBW2Lq8NoYaFqaHclyuFJ9AZv
XuP5FonjmRXgYBDmpTMuKhEZn4KZ2ogwBdrcAcH/FbaHrnDQLMlRAGMAvfLA30Ub
kPzeqQzuBfHLDO04D7op4zkfBJ1+o0/uWMquJK0afjeCe6lNQCaoEAHnTsoZLiVg
2Qny1anOIzHjZxQ1xsIFmE1hc5rlHKqWv1jmULl3XJPj+Bqi8pDfnpc2yQh43WLB
L8bk6lJPLlHUdBW+BIjE
=lPkA
-----END PGP SIGNATURE-----
Fri Oct 28 22:27:25 UTC 2016  I: Checking whether the package is not for us
Fri Oct 28 22:27:25 UTC 2016  I: Starting 1st build on remote node profitbricks-build2-i386.debian.net.
I: using fakeroot in build.
I: pbuilder: network access will be disabled during build
I: Current time: Fri Oct 28 10:27:28 -12 2016
I: pbuilder-time-stamp: 1477693648
I: Building the build Environment
I: extracting base tarball [/var/cache/pbuilder/unstable-reproducible-base.tgz]
I: copying local configuration
I: mounting /proc filesystem
I: mounting /sys filesystem
I: mounting /run/shm filesystem
I: mounting /dev/pts filesystem
I: Mounting /dev/shm
I: policy-rc.d already exists
I: using eatmydata during job
I: Installing the build-deps
 -> Attempting to satisfy build-dependencies
 -> Creating pbuilder-satisfydepends-dummy package
Package: pbuilder-satisfydepends-dummy
Version: 0.invalid.0
Architecture: i386
Maintainer: Debian Pbuilder Team <pbuilder-maint@lists.alioth.debian.org>
Description: Dummy package to satisfy dependencies with aptitude - created by pbuilder
 This package was created automatically by pbuilder to satisfy the
 build-dependencies of the package being currently built.
Depends: debhelper (>= 9), dh-golang, golang-go
dpkg-deb: building package 'pbuilder-satisfydepends-dummy' in '/tmp/satisfydepends-aptitude/pbuilder-satisfydepends-dummy.deb'.
Selecting previously unselected package pbuilder-satisfydepends-dummy.
(Reading database ... 18610 files and directories currently installed.)
Preparing to unpack .../pbuilder-satisfydepends-dummy.deb ...
Unpacking pbuilder-satisfydepends-dummy (0.invalid.0) ...
dpkg: pbuilder-satisfydepends-dummy: dependency problems, but configuring anyway as you requested:
 pbuilder-satisfydepends-dummy depends on debhelper (>= 9); however:
  Package debhelper is not installed.
 pbuilder-satisfydepends-dummy depends on dh-golang; however:
  Package dh-golang is not installed.
 pbuilder-satisfydepends-dummy depends on golang-go; however:
  Package golang-go is not installed.

Setting up pbuilder-satisfydepends-dummy (0.invalid.0) ...
Reading package lists...
Building dependency tree...
Reading state information...
Initializing package states...
Writing extended state information...
Building tag database...
pbuilder-satisfydepends-dummy is already installed at the requested version (0.invalid.0)
pbuilder-satisfydepends-dummy is already installed at the requested version (0.invalid.0)
The following NEW packages will be installed:
  autoconf{a} automake{a} autopoint{a} autotools-dev{a} bsdmainutils{a} debhelper{a} dh-autoreconf{a} dh-golang{a} dh-strip-nondeterminism{a} file{a} gettext{a} gettext-base{a} golang-1.7-go{a} golang-1.7-src{a} golang-go{a} golang-src{a} groff-base{a} intltool-debian{a} libarchive-zip-perl{a} libbsd0{a} libcroco3{a} libffi6{a} libfile-stripnondeterminism-perl{a} libglib2.0-0{a} libicu57{a} libmagic-mgc{a} libmagic1{a} libpipeline1{a} libsigsegv2{a} libtimedate-perl{a} libtool{a} libunistring0{a} libxml2{a} m4{a} man-db{a} po-debconf{a} 
The following packages are RECOMMENDED but will NOT be installed:
  curl libglib2.0-data libltdl-dev libmail-sendmail-perl lynx-cur pkg-config pkgconf shared-mime-info wget xdg-user-dirs xml-core 
0 packages upgraded, 36 newly installed, 0 to remove and 0 not upgraded.
Need to get 45.8 MB of archives. After unpacking 205 MB will be used.
Writing extended state information...
Get: 1 http://ftp.de.debian.org/debian unstable/main i386 groff-base i386 1.22.3-8 [1169 kB]
Get: 2 http://ftp.de.debian.org/debian unstable/main i386 libbsd0 i386 0.8.3-1 [86.5 kB]
Get: 3 http://ftp.de.debian.org/debian unstable/main i386 bsdmainutils i386 9.0.12 [188 kB]
Get: 4 http://ftp.de.debian.org/debian unstable/main i386 libpipeline1 i386 1.4.1-2 [29.6 kB]
Get: 5 http://ftp.de.debian.org/debian unstable/main i386 man-db i386 2.7.5-1 [1011 kB]
Get: 6 http://ftp.de.debian.org/debian unstable/main i386 libmagic-mgc i386 1:5.29-1 [219 kB]
Get: 7 http://ftp.de.debian.org/debian unstable/main i386 libmagic1 i386 1:5.29-1 [116 kB]
Get: 8 http://ftp.de.debian.org/debian unstable/main i386 file i386 1:5.29-1 [63.4 kB]
Get: 9 http://ftp.de.debian.org/debian unstable/main i386 gettext-base i386 0.19.8.1-1 [121 kB]
Get: 10 http://ftp.de.debian.org/debian unstable/main i386 libicu57 i386 57.1-4 [7827 kB]
Get: 11 http://ftp.de.debian.org/debian unstable/main i386 libxml2 i386 2.9.4+dfsg1-2 [953 kB]
Get: 12 http://ftp.de.debian.org/debian unstable/main i386 libsigsegv2 i386 2.10-5 [29.1 kB]
Get: 13 http://ftp.de.debian.org/debian unstable/main i386 m4 i386 1.4.17-5 [255 kB]
Get: 14 http://ftp.de.debian.org/debian unstable/main i386 autoconf all 2.69-10 [338 kB]
Get: 15 http://ftp.de.debian.org/debian unstable/main i386 autotools-dev all 20160430.1 [72.6 kB]
Get: 16 http://ftp.de.debian.org/debian unstable/main i386 automake all 1:1.15-4 [735 kB]
Get: 17 http://ftp.de.debian.org/debian unstable/main i386 autopoint all 0.19.8.1-1 [433 kB]
Get: 18 http://ftp.de.debian.org/debian unstable/main i386 libtool all 2.4.6-2 [545 kB]
Get: 19 http://ftp.de.debian.org/debian unstable/main i386 dh-autoreconf all 12 [15.8 kB]
Get: 20 http://ftp.de.debian.org/debian unstable/main i386 libarchive-zip-perl all 1.59-1 [95.5 kB]
Get: 21 http://ftp.de.debian.org/debian unstable/main i386 libfile-stripnondeterminism-perl all 0.028-1 [14.6 kB]
Get: 22 http://ftp.de.debian.org/debian unstable/main i386 libtimedate-perl all 2.3000-2 [42.2 kB]
Get: 23 http://ftp.de.debian.org/debian unstable/main i386 dh-strip-nondeterminism all 0.028-1 [9020 B]
Get: 24 http://ftp.de.debian.org/debian unstable/main i386 libffi6 i386 3.2.1-6 [19.5 kB]
Get: 25 http://ftp.de.debian.org/debian unstable/main i386 libglib2.0-0 i386 2.50.1-1 [2758 kB]
Get: 26 http://ftp.de.debian.org/debian unstable/main i386 libcroco3 i386 0.6.11-2 [149 kB]
Get: 27 http://ftp.de.debian.org/debian unstable/main i386 libunistring0 i386 0.9.6+really0.9.3-0.1 [285 kB]
Get: 28 http://ftp.de.debian.org/debian unstable/main i386 gettext i386 0.19.8.1-1 [1497 kB]
Get: 29 http://ftp.de.debian.org/debian unstable/main i386 intltool-debian all 0.35.0+20060710.4 [26.3 kB]
Get: 30 http://ftp.de.debian.org/debian unstable/main i386 po-debconf all 1.0.20 [247 kB]
Get: 31 http://ftp.de.debian.org/debian unstable/main i386 debhelper all 10.2.2 [826 kB]
Get: 32 http://ftp.de.debian.org/debian unstable/main i386 golang-1.7-src i386 1.7.3-1 [7647 kB]
Get: 33 http://ftp.de.debian.org/debian unstable/main i386 golang-1.7-go i386 1.7.3-1 [17.9 MB]
Get: 34 http://ftp.de.debian.org/debian unstable/main i386 golang-src i386 2:1.7~1 [3186 B]
Get: 35 http://ftp.de.debian.org/debian unstable/main i386 golang-go i386 2:1.7~1 [22.3 kB]
Get: 36 http://ftp.de.debian.org/debian unstable/main i386 dh-golang all 1.19 [9548 B]
Fetched 45.8 MB in 2s (17.1 MB/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package groff-base.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 18610 files and directories currently installed.)
Preparing to unpack .../00-groff-base_1.22.3-8_i386.deb ...
Unpacking groff-base (1.22.3-8) ...
Selecting previously unselected package libbsd0:i386.
Preparing to unpack .../01-libbsd0_0.8.3-1_i386.deb ...
Unpacking libbsd0:i386 (0.8.3-1) ...
Selecting previously unselected package bsdmainutils.
Preparing to unpack .../02-bsdmainutils_9.0.12_i386.deb ...
Unpacking bsdmainutils (9.0.12) ...
Selecting previously unselected package libpipeline1:i386.
Preparing to unpack .../03-libpipeline1_1.4.1-2_i386.deb ...
Unpacking libpipeline1:i386 (1.4.1-2) ...
Selecting previously unselected package man-db.
Preparing to unpack .../04-man-db_2.7.5-1_i386.deb ...
Unpacking man-db (2.7.5-1) ...
Selecting previously unselected package libmagic-mgc.
Preparing to unpack .../05-libmagic-mgc_1%3a5.29-1_i386.deb ...
Unpacking libmagic-mgc (1:5.29-1) ...
Selecting previously unselected package libmagic1:i386.
Preparing to unpack .../06-libmagic1_1%3a5.29-1_i386.deb ...
Unpacking libmagic1:i386 (1:5.29-1) ...
Selecting previously unselected package file.
Preparing to unpack .../07-file_1%3a5.29-1_i386.deb ...
Unpacking file (1:5.29-1) ...
Selecting previously unselected package gettext-base.
Preparing to unpack .../08-gettext-base_0.19.8.1-1_i386.deb ...
Unpacking gettext-base (0.19.8.1-1) ...
Selecting previously unselected package libicu57:i386.
Preparing to unpack .../09-libicu57_57.1-4_i386.deb ...
Unpacking libicu57:i386 (57.1-4) ...
Selecting previously unselected package libxml2:i386.
Preparing to unpack .../10-libxml2_2.9.4+dfsg1-2_i386.deb ...
Unpacking libxml2:i386 (2.9.4+dfsg1-2) ...
Selecting previously unselected package libsigsegv2:i386.
Preparing to unpack .../11-libsigsegv2_2.10-5_i386.deb ...
Unpacking libsigsegv2:i386 (2.10-5) ...
Selecting previously unselected package m4.
Preparing to unpack .../12-m4_1.4.17-5_i386.deb ...
Unpacking m4 (1.4.17-5) ...
Selecting previously unselected package autoconf.
Preparing to unpack .../13-autoconf_2.69-10_all.deb ...
Unpacking autoconf (2.69-10) ...
Selecting previously unselected package autotools-dev.
Preparing to unpack .../14-autotools-dev_20160430.1_all.deb ...
Unpacking autotools-dev (20160430.1) ...
Selecting previously unselected package automake.
Preparing to unpack .../15-automake_1%3a1.15-4_all.deb ...
Unpacking automake (1:1.15-4) ...
Selecting previously unselected package autopoint.
Preparing to unpack .../16-autopoint_0.19.8.1-1_all.deb ...
Unpacking autopoint (0.19.8.1-1) ...
Selecting previously unselected package libtool.
Preparing to unpack .../17-libtool_2.4.6-2_all.deb ...
Unpacking libtool (2.4.6-2) ...
Selecting previously unselected package dh-autoreconf.
Preparing to unpack .../18-dh-autoreconf_12_all.deb ...
Unpacking dh-autoreconf (12) ...
Selecting previously unselected package libarchive-zip-perl.
Preparing to unpack .../19-libarchive-zip-perl_1.59-1_all.deb ...
Unpacking libarchive-zip-perl (1.59-1) ...
Selecting previously unselected package libfile-stripnondeterminism-perl.
Preparing to unpack .../20-libfile-stripnondeterminism-perl_0.028-1_all.deb ...
Unpacking libfile-stripnondeterminism-perl (0.028-1) ...
Selecting previously unselected package libtimedate-perl.
Preparing to unpack .../21-libtimedate-perl_2.3000-2_all.deb ...
Unpacking libtimedate-perl (2.3000-2) ...
Selecting previously unselected package dh-strip-nondeterminism.
Preparing to unpack .../22-dh-strip-nondeterminism_0.028-1_all.deb ...
Unpacking dh-strip-nondeterminism (0.028-1) ...
Selecting previously unselected package libffi6:i386.
Preparing to unpack .../23-libffi6_3.2.1-6_i386.deb ...
Unpacking libffi6:i386 (3.2.1-6) ...
Selecting previously unselected package libglib2.0-0:i386.
Preparing to unpack .../24-libglib2.0-0_2.50.1-1_i386.deb ...
Unpacking libglib2.0-0:i386 (2.50.1-1) ...
Selecting previously unselected package libcroco3:i386.
Preparing to unpack .../25-libcroco3_0.6.11-2_i386.deb ...
Unpacking libcroco3:i386 (0.6.11-2) ...
Selecting previously unselected package libunistring0:i386.
Preparing to unpack .../26-libunistring0_0.9.6+really0.9.3-0.1_i386.deb ...
Unpacking libunistring0:i386 (0.9.6+really0.9.3-0.1) ...
Selecting previously unselected package gettext.
Preparing to unpack .../27-gettext_0.19.8.1-1_i386.deb ...
Unpacking gettext (0.19.8.1-1) ...
Selecting previously unselected package intltool-debian.
Preparing to unpack .../28-intltool-debian_0.35.0+20060710.4_all.deb ...
Unpacking intltool-debian (0.35.0+20060710.4) ...
Selecting previously unselected package po-debconf.
Preparing to unpack .../29-po-debconf_1.0.20_all.deb ...
Unpacking po-debconf (1.0.20) ...
Selecting previously unselected package debhelper.
Preparing to unpack .../30-debhelper_10.2.2_all.deb ...
Unpacking debhelper (10.2.2) ...
Selecting previously unselected package golang-1.7-src.
Preparing to unpack .../31-golang-1.7-src_1.7.3-1_i386.deb ...
Unpacking golang-1.7-src (1.7.3-1) ...
Selecting previously unselected package golang-1.7-go.
Preparing to unpack .../32-golang-1.7-go_1.7.3-1_i386.deb ...
Unpacking golang-1.7-go (1.7.3-1) ...
Selecting previously unselected package golang-src.
Preparing to unpack .../33-golang-src_2%3a1.7~1_i386.deb ...
Unpacking golang-src (2:1.7~1) ...
Selecting previously unselected package golang-go.
Preparing to unpack .../34-golang-go_2%3a1.7~1_i386.deb ...
Unpacking golang-go (2:1.7~1) ...
Selecting previously unselected package dh-golang.
Preparing to unpack .../35-dh-golang_1.19_all.deb ...
Unpacking dh-golang (1.19) ...
Setting up libarchive-zip-perl (1.59-1) ...
Setting up libtimedate-perl (2.3000-2) ...
Setting up libsigsegv2:i386 (2.10-5) ...
Setting up groff-base (1.22.3-8) ...
Setting up gettext-base (0.19.8.1-1) ...
Setting up libpipeline1:i386 (1.4.1-2) ...
Setting up m4 (1.4.17-5) ...
Setting up libicu57:i386 (57.1-4) ...
Setting up libbsd0:i386 (0.8.3-1) ...
Setting up libxml2:i386 (2.9.4+dfsg1-2) ...
Setting up libmagic-mgc (1:5.29-1) ...
Setting up libmagic1:i386 (1:5.29-1) ...
Processing triggers for libc-bin (2.24-5) ...
Setting up autotools-dev (20160430.1) ...
Setting up libunistring0:i386 (0.9.6+really0.9.3-0.1) ...
Setting up libffi6:i386 (3.2.1-6) ...
Setting up golang-1.7-src (1.7.3-1) ...
Setting up bsdmainutils (9.0.12) ...
update-alternatives: using /usr/bin/bsd-write to provide /usr/bin/write (write) in auto mode
update-alternatives: using /usr/bin/bsd-from to provide /usr/bin/from (from) in auto mode
Setting up autopoint (0.19.8.1-1) ...
Setting up libfile-stripnondeterminism-perl (0.028-1) ...
Setting up golang-src (2:1.7~1) ...
Setting up libglib2.0-0:i386 (2.50.1-1) ...
No schema files found: doing nothing.
Setting up autoconf (2.69-10) ...
Setting up file (1:5.29-1) ...
Setting up libcroco3:i386 (0.6.11-2) ...
Setting up golang-1.7-go (1.7.3-1) ...
Setting up automake (1:1.15-4) ...
update-alternatives: using /usr/bin/automake-1.15 to provide /usr/bin/automake (automake) in auto mode
Setting up man-db (2.7.5-1) ...
Building database of manual pages ...
Setting up libtool (2.4.6-2) ...
Setting up gettext (0.19.8.1-1) ...
Setting up golang-go (2:1.7~1) ...
Setting up intltool-debian (0.35.0+20060710.4) ...
Setting up po-debconf (1.0.20) ...
Setting up dh-autoreconf (12) ...
Setting up dh-strip-nondeterminism (0.028-1) ...
Setting up debhelper (10.2.2) ...
Setting up dh-golang (1.19) ...
Processing triggers for libc-bin (2.24-5) ...
Reading package lists...
Building dependency tree...
Reading state information...
Reading extended state information...
Initializing package states...
Writing extended state information...
Building tag database...
 -> Finished parsing the build-deps
Reading package lists...
Building dependency tree...
Reading state information...
fakeroot is already the newest version (1.21-2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
I: Copying source file
I: copying [golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.dsc]
I: copying [./golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28.orig.tar.xz]
I: copying [./golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.debian.tar.xz]
I: Extracting source
gpgv: unknown type of key resource 'trustedkeys.kbx'
gpgv: keyblock resource '/root/.gnupg/trustedkeys.kbx': General error
gpgv: Signature made Tue Oct 25 01:43:41 2016 -12
gpgv:                using RSA key E8D37AE2F09F4872
gpgv: Can't check signature: No public key
dpkg-source: warning: failed to verify signature on ./golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.dsc
dpkg-source: info: extracting golang-github-hydrogen18-stalecucumber in golang-github-hydrogen18-stalecucumber-0.0~git20161012.0.cd9ec28
dpkg-source: info: unpacking golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28.orig.tar.xz
dpkg-source: info: unpacking golang-github-hydrogen18-stalecucumber_0.0~git20161012.0.cd9ec28-1.debian.tar.xz
I: Building the package
I: Running cd /build-1st/*/ && env PATH="/usr/sbin:/usr/bin:/sbin:/bin:/usr/games" HOME="/nonexistent/first-build" dpkg-buildpackage -us -uc -b --buildinfo-id=i386 -rfakeroot
dpkg-buildpackage: info: source package golang-github-hydrogen18-stalecucumber
dpkg-buildpackage: info: source version 0.0~git20161012.0.cd9ec28-1
dpkg-buildpackage: info: source distribution unstable
dpkg-buildpackage: info: source changed by Sascha Steinbiss <satta@debian.org>
 dpkg-source --before-build golang-github-hydrogen18-stalecucumber-0.0~git20161012.0.cd9ec28
dpkg-buildpackage: info: host architecture i386
 fakeroot debian/rules clean
dh clean --buildsystem=golang --with=golang
   dh_testdir -O--buildsystem=golang
   dh_auto_clean -O--buildsystem=golang
   dh_clean -O--buildsystem=golang
 debian/rules build
dh build --buildsystem=golang --with=golang
   dh_testdir -O--buildsystem=golang
   dh_update_autotools_config -O--buildsystem=golang
   dh_auto_configure -O--buildsystem=golang
   dh_auto_build -O--buildsystem=golang
	go install -v -p 1 github.com/hydrogen18/stalecucumber github.com/hydrogen18/stalecucumber/struct_export_test
github.com/hydrogen18/stalecucumber
github.com/hydrogen18/stalecucumber/struct_export_test
   dh_auto_test -O--buildsystem=golang
	go test -v -p 1 github.com/hydrogen18/stalecucumber github.com/hydrogen18/stalecucumber/struct_export_test
# github.com/hydrogen18/stalecucumber
src/github.com/hydrogen18/stalecucumber/pickle_writer_test.go:132: constant 18446744073709551615 overflows uint
FAIL	github.com/hydrogen18/stalecucumber [build failed]
?   	github.com/hydrogen18/stalecucumber/struct_export_test	[no test files]
dh_auto_test: go test -v -p 1 github.com/hydrogen18/stalecucumber github.com/hydrogen18/stalecucumber/struct_export_test returned exit code 2
debian/rules:4: recipe for target 'build' failed
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
I: copying local configuration
E: Failed autobuilding of package
I: unmounting /dev/shm filesystem
I: unmounting dev/pts filesystem
I: unmounting run/shm filesystem
I: unmounting proc filesystem
I: unmounting sys filesystem
I: cleaning the build env 
I: removing directory /srv/workspace/pbuilder/30862 and its subdirectories
Fri Oct 28 22:30:25 UTC 2016 - total duration: 0h 3m 14s.

--- a/pickle_writer_test.go
+++ b/pickle_writer_test.go
@@ -129,7 +129,7 @@ func TestPickleUint64(t *testing.T) {
 	inAndOut(i, bi, t)
 
 	var ui uint
-	ui = 18446744073709551615
+	ui = ^uint(0)
 	bi.SetUint64(uint64(ui))
 	inAndOut(ui, bi, t)
 }

